src/extractor/extraction_containers.cpp

```diff
@@ -795,6 +795,41 @@ void ExtractionContainers::PrepareManeuverOverrides()
                 from_segment_itr->second, to_segment_itr->second, via_node);
         };
 
+    const auto string_to_turn_type = [](const std::string &turn_string) {
+        if (turn_string == "uturn") {
+            return guidance::TurnType::Turn;
+        } else if (turn_string == "continue") {
+            return guidance::TurnType::Continue;
+        } else if (turn_string == "turn") {
+            return guidance::TurnType::Turn;
+        } else if (turn_string == "fork") {
+            return guidance::TurnType::Fork;
+        } else if (turn_string == "suppress") {
+            return guidance::TurnType::Suppressed;
+        } else {
+            return guidance::TurnType::Invalid;
+        }
+    };
+
+    const auto string_to_turn_direction = [](const std::string &direction_string) {
```

Comment:
I think here we will also need to pass in the `turn_string` to be able to set a `uturn` direction modifier